portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeLoginNav.vm

```diff
@@ -79,15 +79,15 @@
                                     #if (${tabsSites.mrphs_profileToolUrl})
 
                                         <div id="loginUser" class="has-avatar Mrphs-userNav__submenuitem--userlink #if (${site.isCurrentSite}) current-site #end">
-                                            <a href="javascript:void(0);" role="button" aria-haspopup="true" class="Mrphs-userNav__drop-btn Mrphs-userNav__submenuitem--username">
+                                            <a href="javascript:void(0);" role="button" title="${loginUserFirstName} ${rloader.sit_profile_drop_down}" aria-haspopup="true" class="Mrphs-userNav__drop-btn Mrphs-userNav__submenuitem--username">
                                                 <span class="Mrphs-userNav__submenuitem--profilepicture" style="background-image:url(/direct/profile/${loginUserId}/image/thumb)"></span>
                                                 <span class="textlink">${loginUserFirstName}</span>
                                             </a>
                                         </div>
 
                                     #else
 
-                                        <a id="loginUser" href="javascript:void(0);" class="Mrphs-userNav__drop-btn no-avatar Mrphs-userNav__submenuitem--userlink #if (${site.isCurrentSite}) current-site #end">
+                                        <a id="loginUser" href="javascript:void(0);" role="menuitem" title="${loginUserFirstName} ${rloader.sit_profile_drop_down}"  aria-haspopup="true" class="Mrphs-userNav__drop-btn no-avatar Mrphs-userNav__submenuitem--userlink #if (${site.isCurrentSite}) current-site #end">
```

Comment:
```suggestion
                                        <a id="loginUser" href="javascript:void(0);" role="menuitem" title="${loginUserFirstName} ${rloader.sit_profile_drop_down}" aria-label="${loginUserFirstName} ${rloader.sit_profile_drop_down}"  aria-haspopup="true" class="Mrphs-userNav__drop-btn no-avatar Mrphs-userNav__submenuitem--userlink #if (${site.isCurrentSite}) current-site #end">
```

Reply:
Adrian - If you look at the includeLoginNav.vm file - there are lots of links that don't follow your practice.   Should I fix all of them?